### ISSUE TITLE
Anti-aliasing on the tree

### ISSUE THREAD

**Issue by nikolaydyankov** [Openarl/PathOfBuilding#426]
Compared to the other passive tree builder (https://github.com/EmmittJ/PoESkillTree) anti-aliasing on the passive tree is something that is really noticeable, especially when you zoom out. Are there plans to add this?

Cheers

**Comment by Openarl** [comment]
The issue isn't lack of anti-aliasing, but rather that the textures used for the lines connecting nodes become too thin when you zoom out beyond a certain point, and no amount of AA or texture filtering will make it look good, as the skill tree just isn't designed to be zoomed out that far. 

PoESkillTree doesn't have this issue because it doesn't use those textures to render the connectors, and that's really the only way to solve the problem. I much prefer using the original artwork though, but I may someday add the option to use solid lines like PoESkillTree does.

**Comment by Lothrik** [comment]
Honestly it looks worse to me with mipmapping enabled, it just seems excessively blurry now.

**Comment by Openarl** [comment]
Yeah, mipmapping's not too good for fine detail. I think it still looks fine up to a certain point though, but as I said before, ultimately the tree's artwork just isn't designed to be viewed like that. I'll leave it as is for the time being and see what other feedback I get.

**Comment by Lothrik** [comment]
I'm not sure it's a mipmapping-specific problem, the artwork resolution is just too low.

For example, on PoB Necromantic Aegis looks like this currently: http://puu.sh/wyV1K/1b51737a95.png
While the actual keystone is much higher resolution, and nowhere near as blurry: http://puu.sh/wyV4s/bc9a76b902.png

**Comment by Openarl** [comment]
That can't be helped; the program uses the assets made by GGG for their online passive tree, and I'm already using the highest resolution version of those assets. I would like to eventually migrate towards using the actual assets and data used by the game itself, but that's no small task.

**Comment by Cephel** [comment]
The lines disappear on some zoom levels too. Making them a minimum width of 1 pixel would solve this.
